appendix.fa.md

```diff
@@ -0,0 +1,39 @@
+RMS has a history of mistreating women and making them feel uncomfortable, unsafe, and unwelcome. For incidents relating to RMS and MIT, please see [0].
```

Comment:
This file can be removed since it has not been translated, and we're linking to the English.

Reply:
Ok, file has been removed.